app/commands/v2/publish.rb

```diff
@@ -12,7 +12,7 @@ def call
     private
 
       def publish_edition
-        delete_change_notes unless update_type == "major"
+        delete_change_notes unless update_type == "major" || update_type == "content_block"
```

Comment:
I think this is fine for now but if this sets a precedent we might want something like
```
UNDELETABLE_CHANGE_NOTE_UPDATE_TYPES = %w(major content_block).freeze
...
delete_change_notes unless UNDELETABLE_CHANGE_NOTE_UPDATE_TYPES.include(update_type)
```

naming things is hard🙈

Reply:
this is a good point, I meant to come back to this line as I didn't like magic strings floating around. I've called the list `UPDATE_TYPES_WITH_CHANGE_NOTES` but not sure if that's a better name :)

Reply:
done! have put at top of private block to keep the scope limited